app/journeys/probate.js

```diff
@@ -160,9 +163,17 @@ const stepList = {
     CopiesOverseas: 'CopiesSummary',
     CopiesSummary: 'TaskList',
 
+    // Payment Task -----------------------------------------------------------
```

Comment:
We don't use comments in the code as the code should be easy to read. Can you remove these comments.